Dalamud/Game/Gui/FlyTextGui.cs

```diff
@@ -0,0 +1,312 @@
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+
+using Dalamud.Game.Text.SeStringHandling;
+using Dalamud.Hooking;
+using FFXIVClientStructs.FFXIV.Client.UI;
+using Serilog;
+
+namespace Dalamud.Game.Gui
```

Comment:
I meant add another `FlyText` to the namespace, something like that anyways. Anything past 2 files gets a namespace. `Dalamud.Game.Gui.FlyText.FlyTextGui`

Reply:
Moved 🙂